### ISSUE TITLE
undefined method `emit' for #<Fluent::Plugin::CopyOutput>

### ISSUE THREAD

**Issue by calexicoz** [tagomoris/fluent-plugin-forest#30]
Hi,

I recently installed the most recent version of Fluentd and the Forest plugin, and I got this in my logs:

```
emit transaction failed: error_class=NoMethodError error="undefined method `emit' for #<Fluent::Plugin::CopyOutput>" tag=".."
/var/lib/gems/2.3.0/gems/fluent-plugin-forest-0.3.0/lib/fluent/plugin/out_forest.rb:171:in `emit'¬
/var/lib/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/compat/output.rb:170:in `process'¬
/var/lib/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/plugin/bare_output.rb:53:in `emit_sync'¬
/var/lib/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/event_router.rb:90:in `emit_stream'¬
/var/lib/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/event_router.rb:81:in `emit'¬
/var/lib/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/plugin/in_forward.rb:205:in `on_message'¬
/var/lib/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/plugin/in_forward.rb:295:in `call'¬
/var/lib/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/plugin/in_forward.rb:295:in `block in on_read_msgpack'¬
/var/lib/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/plugin/in_forward.rb:294:in `feed_each'¬
/var/lib/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/plugin/in_forward.rb:294:in `on_read_msgpack'¬
/var/lib/gems/2.3.0/gems/cool.io-1.4.4/lib/cool.io/io.rb:123:in `on_readable'¬
/var/lib/gems/2.3.0/gems/cool.io-1.4.4/lib/cool.io/io.rb:186:in `on_readable'¬
/var/lib/gems/2.3.0/gems/cool.io-1.4.4/lib/cool.io/loop.rb:88:in `run_once'¬
/var/lib/gems/2.3.0/gems/cool.io-1.4.4/lib/cool.io/loop.rb:88:in `run'¬
/var/lib/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/plugin/in_forward.rb:117:in `run'¬
```

Maybe a change introduced in Fluent 0.14?

Thanks.

**Comment by tagomoris** [comment]
Right. I'll fix it later.

**Comment by tagomoris** [comment]
#31 fixes this issue, and I released `v0.3.1` with that fix.
Thank you for reporting!

**Comment by calexicoz** [comment]
Thanks!
